File: Golang/AddBinary/AddBinary_test.go
package AddBinary

import "testing"

func TestAddBinary(t *testing.T) {

	if addBinary("0", "0") == "0" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if addBinary("11", "1") == "100" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if addBinary("101", "101") == "1010" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}

File: Python/AddBinaryTest.py
from unittest import TestCase

from AddBinary import AddBinary


class AddBinaryTest(TestCase):
    def test_addBinary(self):
        ab = AddBinary()

        self.assertTrue(ab.addBinary("0", "0") == "0")

        self.assertTrue(ab.addBinary("11", "1") == "100")

        self.assertTrue(ab.addBinary("101", "101") == "1010")
